src/support/div-by-const.cpp

```diff
@@ -0,0 +1,107 @@
+// Copyright 2014 the V8 project authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
```

Comment:
this LICENSE file should be included. if it's very short, maybe it can just be embedded here. otherwise, it should be a file on the side, which means probably there should be a new subdir for this, so it does not appear to apply to any other files than it should.

Reply:
Sure, make sense. I just wanted to clarify this nuance.
Done!